src/frontend/package.json

```diff
@@ -105,6 +105,6 @@
     "zustand": "3.2.0"
   },
   "resolutions": {
-    "@videojs/http-streaming/mux.js": "5.7.0"
+    "**/mux.js": "5.7.0"
```

Comment:
To only enforce video.js to use mux.js@5.7.0, you could use `"video.js/**/mux.js": "5.7.0"`.